### PR TITLE
Update hololens-cortana.md

### PR DESCRIPTION
**Pull request by tmlyon**
Add information about the online speech recognition setting.

### CODE REVIEW

**Comment by tmlyon** [comment]
@Teresa-Motiv for review

File: devices/hololens/hololens-cortana.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This article teaches you how to control HoloLens and your holographic world with
 
 ## Built-in voice commands
 
-Get around HoloLens faster with these basic commands. In order to use these you need to enable Speech during first run of the device or in **Settings** > **Privacy** > **Speech**. You can always check whether speech is enabled by looking at the status at the top of Start menu.
+Get around HoloLens faster with these basic commands. In order to use these you need to enable Speech during first run of the device or in **Settings** > **Privacy** > **Speech**. You can always check whether speech is enabled by looking at the status at the top of Start menu. For the best recognition of speech commands, HoloLens 2 will use the Microsoft cloud-bases services, but you can also choose to disable this in Settings under **Online speech recogntion** to only run locally instead.
```
Comment (illfated) [inline devices/hololens/hololens-cortana.md]: minor readability improvement (commas)
```suggestion
Get around HoloLens faster with these basic commands. In order to use these you need to enable Speech during first run of the device or in **Settings** > **Privacy** > **Speech**. You can always check whether speech is enabled by looking at the status at the top of Start menu. For the best recognition of speech commands, HoloLens 2 will use the Microsoft cloud-bases services, but you can also choose to disable this in Settings, under **Online speech recogntion**, to only run locally instead.
```

Comment (illfated) [reply devices/hololens/hololens-cortana.md]: (Feel free to ignore (resolve) this suggestion if you find it to be excessive use of commas.)

Comment (tmlyon) [reply devices/hololens/hololens-cortana.md]: Cool with me.

File: devices/hololens/hololens-cortana.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ This article teaches you how to control HoloLens and your holographic world with
 
 ## Built-in voice commands
 
-Get around HoloLens faster with these basic commands. In order to use these you need to enable Speech during first run of the device or in **Settings** > **Privacy** > **Speech**. You can always check whether speech is enabled by looking at the status at the top of Start menu.
+Get around HoloLens faster with these basic commands. In order to use these you need to enable Speech during first run of the device or in **Settings** > **Privacy** > **Speech**. You can always check whether speech is enabled by looking at the status at the top of Start menu. For the best recognition of speech commands, HoloLens 2 will use the Microsoft cloud-bases services, but you can also choose to disable this in Settings, under **Online speech recogntion**, to only run locally instead.
```
Comment (Teresa-Motiv) [inline devices/hololens/hololens-cortana.md]: Revise:
For the best speech recognition results, HoloLens 2 uses the Microsoft cloud-based services. However, you can use Settings to disable this feature. to do this, in Settings, turn off **Online speech recognition**. After you change this setting, HoloLens 2 uses only local data to support voice recognition.

Comment (tmlyon) [reply devices/hololens/hololens-cortana.md]: Proposed additional revision: "**T**o do this, in Settings, turn off Online speech recognition. After you change this setting, HoloLens 2 will only process voice data locally to recognize commands and dictation and Cortana will not be available."

Comment (tmlyon) [reply devices/hololens/hololens-cortana.md]: Hi @Teresa-Motiv, how would you like to close out this issue? I need to make a few additional updates for the Vb speech command changes and would like to merge this in first. Thanks!

Comment (Teresa-Motiv) [reply devices/hololens/hololens-cortana.md:33]: Sorry for the delay, I missed the notification on this one. I don't have permissions to merge PRs in this repo.

Comment (illfated) [reply devices/hololens/hololens-cortana.md:33]: @tmlyon : Would you like to add the suggested change, either your own suggestion or the one from @Teresa-Motiv , before this PR gets its final review & merge? If you find it troublesome to make the change, I could easily add the suggestion directly in your PR to let you merge the suggestion. If so, just tell me whether you want the assistance or not.

Comment (tmlyon) [reply devices/hololens/hololens-cortana.md]: @illfated Just made the changes. Still a bit new to using github... Thanks!

File: devices/hololens/hololens-cortana.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ This article teaches you how to control HoloLens and your holographic world with
 
 ## Built-in voice commands
 
-Get around HoloLens faster with these basic commands. In order to use these you need to enable Speech during first run of the device or in **Settings** > **Privacy** > **Speech**. You can always check whether speech is enabled by looking at the status at the top of Start menu.
+Get around HoloLens faster with these basic commands. In order to use these you need to enable Speech during first run of the device or in **Settings** > **Privacy** > **Speech**. You can always check whether speech is enabled by looking at the status at the top of Start menu. For the best speech recognition results, HoloLens 2 uses the Microsoft cloud-based services. However, you can use Settings to disable this feature. To do this, in Settings, turn off *Online speech recognition*. After you change this setting, HoloLens 2 will only process voice data locally to recognize commands and dictation and Cortana will not be available.
```
Comment (illfated) [inline devices/hololens/hololens-cortana.md:33]: Readability commas and particles (minor edits):
```suggestion
Get around HoloLens faster with these basic commands. In order to use these, you need to enable Speech during the first run of the device or in **Settings** > **Privacy** > **Speech**. You can always check whether speech is enabled by looking at the status at the top of the Start menu. For the best speech recognition results, HoloLens 2 uses the Microsoft cloud-based services. However, you can use Settings to disable this feature. To do this, in Settings, turn off *Online speech recognition*. After you change this setting, HoloLens 2 will only process voice data locally to recognize commands and dictation, and Cortana will not be available.
```

**Comment by tmlyon** [comment]
Fyi to @TinaMcN  since this is also a pending edit to a specially localized page.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**